convert2rhel/backup.py

```diff
@@ -374,6 +390,21 @@ def enable(self):
         # Set the enabled value
         super(NewRestorableFile, self).enable()
 
+    def _normalize_backup_path(self):
```

Comment:
I find the use of 'normalize' here confusing; in the Python world, 'normalize' refers to what `os.path.normpath` does.

Reply:
Agreed. Now reading it over again, it really seems a bit confusing.